sql/moz-fx-data-marketing-prod/ga_derived/www_site_downloads_v1/query.sql

```diff
@@ -35,7 +35,7 @@ WITH download_event_count AS (
 SELECT
   *,
   COUNTIF(download_events > 0) AS downloads,
-  COUNTIF(download_events > 0 AND browser != 'Firefox') AS non_fx_downloads,
+  COUNTIF(download_events > 0 AND NOT udf.ga_is_mozilla_browser(browser)) AS non_fx_downloads,
```

Comment:
query runs in a different project and I think it makes sense to keep the udf in shared-prod with everything else
```suggestion
  COUNTIF(download_events > 0 AND NOT `moz-fx-data-shared-prod.udf.ga_is_mozilla_browser`(browser)) AS non_fx_downloads,
```

Reply:
I realize this is a confusing side effect of how the projects and bigquery-etl is set up